### ISSUE TITLE
[tool_crash] NoSuchMethodError: The getter 'type' was called on null

### ISSUE THREAD

**Issue by shinsenter** [flutter/flutter#67384]
## Command
```
flutter build ios --obfuscate --release --analyze-size --split-debug-info=build/debug_info
```

## Steps to Reproduce
1. Run build command with --analyze-size

## Logs
NoSuchMethodError: NoSuchMethodError: The getter 'type' was called on null.
Receiver: null
Tried calling: type
```
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
#1      _ProgramInfoBuilder.createInfoNodeFor (package:vm_snapshot_analysis/v8_profile.dart:415:29)
#2      _ProgramInfoBuilder.getInfoNodeFor (package:vm_snapshot_analysis/v8_profile.dart:350:14)
#3      ListIterable.forEach (dart:_internal/iterable.dart:39:13)
#4      _ProgramInfoBuilder.build (package:vm_snapshot_analysis/v8_profile.dart:314:16)
#5      toProgramInfo (package:vm_snapshot_analysis/v8_profile.dart:258:8)
#6      _treemapFromSnapshot (package:vm_snapshot_analysis/treemap.dart:157:16)
#7      treemapFromJson (package:vm_snapshot_analysis/treemap.dart:85:5)
#8      SizeAnalyzer.analyzeAotSnapshot (package:flutter_tools/src/base/analyze_size.dart:57:59)
#9      BuildIOSCommand.runCommand (package:flutter_tools/src/commands/build_ios.dart:143:61)
#10     _rootRunUnary (dart:async/zone.dart:1198:47)
#11     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
#12     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
#13     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
#14     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
#15     Future._completeWithValue (dart:async/future_impl.dart:529:5)
#16     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
#17     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
#18     buildXcodeProject (package:flutter_tools/src/ios/mac.dart)
#19     _rootRunUnary (dart:async/zone.dart:1198:47)
#20     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
#21     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
#22     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
#23     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
#24     Future._completeWithValue (dart:async/future_impl.dart:529:5)
```
```
[✓] Flutter (Channel stable, 1.22.0, on macOS 11.0 20A5384c, locale ja)
    • Flutter version 1.22.0 at /Users/develop/flutter
    • Framework revision d408d302e2 (7 days ago), 2020-09-29 11:49:17 -0700
    • Engine revision 5babba6c4d
    • Dart version 2.10.0

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/webike/Library/Android/sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0.1, Build version 12A7300
    • CocoaPods version 1.10.0.rc.1

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.49.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.15.0

[✓] Connected device (2 available)
    • MAR LX2J (mobile) • STP0220108003514                         • android-arm64 • Android 10 (API 29)
    • iPhone (mobile)   • bc2bdad813b206d48493be94d4109105585b5496 • ios           • iOS 12.4.8
    ! Error: iPhone is busy: Copying cache files from device. Xcode will continue when iPhone is finished. (code -10)
    ! Error: iPhone Xtream is not connected. Xcode will continue when iPhone Xtream is connected. (code -13)

! Doctor found issues in 1 category.

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.7.1+76
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: stable
**Creation framework version**: 20e59316b8b8474554b38493b8ca888794b0234a
### Plugins
app_review-2.0.1
device_info-0.4.2+8
firebase_analytics-6.0.1
firebase_analytics_web-0.1.1
firebase_core-0.5.0
firebase_core_web-0.2.0
firebase_crashlytics-0.2.1
firebase_messaging-7.0.2
firebase_performance-0.4.0+1
flutter_app_badger-1.1.2
flutter_inappwebview-4.0.0+4
flutter_jailbreak_detection-1.4.0
flutter_plugin_android_lifecycle-1.0.11
geolocator-6.0.0+4
google_maps_flutter-1.0.2
map_launcher-0.12.2
package_info-0.4.3
path_provider-1.6.18
path_provider_linux-0.0.1+2
path_provider_macos-0.0.4+4
path_provider_windows-0.0.4+1
permission_handler-5.0.1+1
screen-0.0.5
shared_preferences-0.5.12
shared_preferences_linux-0.0.2+2
shared_preferences_macos-0.0.1+10
shared_preferences_web-0.1.2+7
shared_preferences_windows-0.0.1+1
sqflite-1.3.1+1
uni_links-0.4.0
url_launcher-5.7.2
url_launcher_linux-0.0.1+1
url_launcher_macos-0.0.1+8
url_launcher_web-0.1.4+1
url_launcher_windows-0.0.1+1
vibration-1.7.0
vibration_web-1.6.1
webview_flutter-1.0.0
qr_code_scanner

**Comment by sidrao2006** [comment]
Duplicate of #66962
Fixed by #66983

**Comment by jonahwilliams** [comment]
^ This is correct, closing
